### ISSUE TITLE
Configuration issues

### ISSUE THREAD

**Issue by Roee1989** [rakehsaleem/Crack-Detection-TF-1.x.x#3]
Hi,
I just came across your project and your weight (mask_rcnn_crack.h5) seem to work fine with my data.
I wanted to know:
Did you use the default RPN_ANCHOR_SCALES?
Did you use the default LOSS_WEIGHTS?
What other parameters did you change?
Did you apply any augmentations?
How many epochs did you train for?
In general, I want to fine tune your weights to my data but I'm getting lost in the way....

**Comment by Roee1989** [comment]
![image](https://user-images.githubusercontent.com/68921679/168838327-0e6ac0f7-7a93-43b0-8586-1964b49580cd.png)
![image](https://user-images.githubusercontent.com/68921679/168838999-84c5a8b1-9bac-458f-97ce-f02b1ce4e475.png)
Those are 2/170 training images I have with your weights applied.
The following image is my weights (50 epochs with RPN_ANCHOR_SCALES = (128, 256, 512, 1024, 1216) and RPN_ANCHOR_RATIOS = [0.25, 1, 4])
![image](https://user-images.githubusercontent.com/68921679/168839925-59fe700a-de19-4cd8-8d3b-1512e4f5562a.png)

**Comment by rakehsaleem** [comment]
Hi @Roee1989 , thanks for reaching out.
I used the default settings for LOSS_WEIGHTS AND RPN_ANCHOR_SCALES, however, I did change some parameters depending on the problem. I applied data augmentations to increase my dataset, and I tried to train for around 500 epochs with a steps/epoch to be 800~1000. Other parameters are the same or default in the config files while learning rate and drop out is something you need to see for your problem. Start with less expensive parameters so the computation is not expensive and then gradually change the parameters depending on the performance of the network. You can visualize the training using tensorboard graph and proceed from there. Hope it helps!! Thank you.

**Comment by Roee1989** [comment]
Thanks for the tips.
I managed to get good enough results with 100 steps/epoch and 50 epochs
with an erosion post processing.
I'll definitely try your suggestions.

‫בתאריך יום ה׳, 23 ביוני 2022 ב-22:27 מאת ‪Rakeh Saleem‬‏ <‪
***@***.***‬‏>:‬
